clients/stellar-relay-lib/src/connection/flow_controller.rs

```diff
@@ -1,42 +1,101 @@
-use substrate_stellar_sdk::types::MessageType;
+use substrate_stellar_sdk::types::{MessageType, SendMore, SendMoreExtended, StellarMessage};
 
-pub const MAX_FLOOD_MSG_CAP: u32 = 2000;
+pub const MAX_FLOOD_MSG_CAP: u32 = 200;
+pub const FLOW_CONTROL_SEND_MORE_BATCH_SIZE: u32 = 40;
+pub const PER_FLOOD_READING_CAPACITY_BYTES: u32 = 300000;
+pub const FLOW_CONTROL_SEND_MORE_BATCH_SIZE_BYTES: u32 = 100000;
 
 #[derive(Debug, Default)]
 pub struct FlowController {
-	enabled: bool,
-	flood_msg_cap: u32,
+	pub(crate) flow_control_bytes_enabled: bool,
```

Comment:
Does this have to be accessible anywhere else, inside the crate?

Reply:
Actually now that you say, it doesn't. I did it because I was [trying](https://github.com/pendulum-chain/spacewalk/pull/550#discussion_r1731079342) to re-add a test for it but I found some roadblocks for that.